include/lsst/afw/detection/Footprint.h

```diff
@@ -251,38 +178,12 @@ class Footprint : public lsst::daf::base::Citizen,
      * region: Footprint's region (default: getRegion())
      */
     template<typename PixelT>
-    void insertIntoImage(typename lsst::afw::image::Image<PixelT>& idImage,
+    void insertIntoImage(typename image::Image<PixelT>& idImage,
                          std::uint64_t const id,
                          geom::Box2I const& region=geom::Box2I()
     ) const;
 
     /**
-     * Set the pixels in idImage which are in Footprint by adding the
-     * specified value to the Image.
-     *
-     * The list of ids found under the new Footprint are returned.
-     *
-     * idImage: Image to contain the footprint
-     * id: Add id to idImage for pixels in the Footprint
-     * overwriteId: should id replace any value already in idImage?
-     * idMask: Don't overwrite ID bits in this mask
-     * oldIds: if non-NULL, set the IDs that were overwritten
-     * region: Footprint's region (default: getRegion())
-     */
-    template<typename PixelT>
-    void insertIntoImage(typename lsst::afw::image::Image<PixelT>& idImage,
-                         std::uint64_t const id,
-                         bool const overwriteId, long const idMask,
-                         typename std::set<std::uint64_t> *oldIds,
-                         geom::Box2I const& region=geom::Box2I()
-    ) const;
```

Comment:
Please check, but I think it was probably used only in merging footprints (the uint64 is needed as we needed to keep two uint32 numbers from conflicting, although I now forget why I needed uint32 not uint16).